test/performance/benchmarks/rollout-probe/continuous/main.go

```diff
@@ -126,7 +127,24 @@ LOOP:
 			if err != nil {
 				log.Fatalf("Error getting ksvc %s: %v", *target, err)
 			}
+			svc = svc.DeepCopy()
 			// Make sure we start with a single instance.
+
+			// At the end of the benchmark, restore to the previous value.
+			if prev := svc.Spec.Template.Annotations["autoscaling.knative.dev/minScale"]; prev != "" {
+				defer func() {
```

Comment:
Do you mean for this to be function scoped? 🤔

Reply:
this select is executed once, so should be ok